android/gltfio-android/src/main/java/com/google/android/filament/gltfio/AssetLoader.java

```diff
@@ -37,7 +37,6 @@
  *
  * companion object {
  *     init {
- *        Filament.init()
```

Comment:
According to the documentation you should not call `Filament.init` if calling `AssetLoader.init` so I removed it.

Reply:
Thanks!